project/Common.scala

```diff
@@ -2,8 +2,8 @@
 
 object Common {
   def organization = "com.iterable"
-  def version = "3.0.0-M10"
-  def playVersion = System.getProperty("play.version", "2.6.12")
-  def scalaVersion = System.getProperty("scala.version", "2.12.6")
+  def version = "4.0.0-M1"
+  def playVersion = System.getProperty("play.version", "2.7.0")
```

Comment:
jaliss/securesocial is still on 2.6.12, does this actually just work without issue?

Reply:
The code seems to build and test just fine, and the samples work, so I don't think we need any other changes. The external libraries we use are still compatible with Play 2.7.